inspire_classifier/config.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Classifier Configuration."""
+
+
+CLASSIFIER_MAXIMUM_VOCABULARY_SIZE = 60000
+CLASSIFIER_MINIMUM_WORD_FREQUENCY = 2
+CLASSIFIER_VALIDATION_DATA_FRACTION = 0.1
+CLASSIFIER_CLASSIFICATION_CLASSES = ['rejected', 'non_core', 'core']
```

Comment:
`non-core`. Besides, to what extent is this configurable Can I change the values here and will everything still work? or do the serializers expect something specific?

Reply:
Do you suggest "non-core" due to compatibility with the previous classifier? If not, I think it might be better to have "non_core" for ease in naming variables and to keep it consistent throughout the application. 
Second, the serializers do expect something specific. But let me comment on that on your other comment.